recipes/coin-utils/all/conanfile.py

```diff
@@ -130,17 +132,17 @@ def package(self):
             autotools = self._configure_autotools()
             autotools.install(args=["-j1"])
 
-        tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.la")
-        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
-        tools.rmdir(os.path.join(self.package_folder, "share"))
+        rm(self, "*.la", f"{self.package_folder}/lib")
+        rmdir(self, f"{self.package_folder}/lib/pkgconfig")
+        rmdir(self, f"{self.package_folder}/share")
 
         if self.settings.compiler == "Visual Studio":
-            os.rename(os.path.join(self.package_folder, "lib", "libCoinUtils.a"),
-                      os.path.join(self.package_folder, "lib", "CoinUtils.lib"))
+            rename(self, f"{self.package_folder}/lib/libCoinUtils.a",
+                         f"{self.package_folder}/lib/CoinUtils.lib")
```

Comment:
seems neater to use use os.path.join, why this change?

Reply:
os.path.join is exactly designed to be cross-platform, why take this risk, we know os.path.join works fine, we don't know yet potential side effects of this change.